components/post_view/post/index.js

```diff
@@ -14,7 +13,6 @@ function mapStateToProps(state, ownProps) {
     const detailedPost = ownProps.post || {};
 
     return {
-        post: getPost(state, detailedPost.id),
```

Comment:
why is this being removed?

Reply:
Not related to this PR, just bumped into this while working on this PR.
I think this is not necessary as it's just going back and forth to the component.

Anyway, I will revert the change and will just deal with it in a separate PR.